pkg/app/web/src/components/application-detail-page/application-detail/index.tsx

```diff
@@ -217,6 +221,14 @@ export const ApplicationDetail: FC<ApplicationDetailProps> = memo(
                 {env.name}
               </Typography>
             )}
+            {app?.labelsMap.map((label, i) => (
+              <Chip
+                label={label[0] + ":" + label[1]}
```

Comment:
How about adding a space between the value and key as `key: value`?

Reply:
For the app list page, you can't do that because the line breaks make it difficult to read. But for the detail page, looks nice:
![image](https://user-images.githubusercontent.com/19730728/149102748-2f6298ff-4fe9-482a-b50a-a2eeebd7186c.png)

Reply:
Applied!

Reply:
Thank you. 👍